pipeline_products_session/jwst-data-products-part2.ipynb

```diff
@@ -0,0 +1,1401 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "An example of a Stage 2 association is shown [in the documentation](https://jwst-pipeline.readthedocs.io/en/latest/jwst/associations/level2_asn_technical.html#example-association), along with a [Stage 3 association](https://jwst-pipeline.readthedocs.io/en/latest/jwst/associations/level3_asn_technical.html#example-association). Unless you are generating your own data or simulations, you will probably not need to create an association file, because you will have the option to retrieve association files from MAST along with your data for reprocessing.  \n",
+    "\n",
+    "However, if you do want to create an association, there are also [command line tools](https://jwst-pipeline.readthedocs.io/en/latest/jwst/associations/asn_from_list.html) included in the pipeline software that help with generating associations for manually running the pipeline. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that that's out of the way, let's continue our data products journey. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Top of Page](#title_ID)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "6.<font color='white'>-</font>Data products: stage 2 (calibrated exposures)<a class=\"anchor\" id=\"stage2\"></a>\n",
+    "------------------\n",
+    "\n",
+    "The paths through the pipeline begin to diverge during Stage 2 for different observing modes. This stage applies physical corrections and calibrations to individual exposures to produce fully calibrated (unrectified) exposures, and the pipeline module used depends on the exposure type: either imaging or spectroscopy. More information can be found in the [JWST User Documentation](https://jwst-docs.stsci.edu/jwst-data-reduction-pipeline/algorithm-documentation/stages-of-processing).\n",
+    "\n",
+    "## 6.1.<font color='white'>-</font>Imaging<a class=\"anchor\" id=\"s2-imaging\"></a>\n",
+    "\n",
+    "Stage 2 image processing applies additional instrumental corrections and calibrations that result in a fully calibrated individual exposure. Non-time series exposures use the CALWEBB_IMAGE2 configuration, which applies all applicable steps to the data. The CALWEBB_TSO-IMAGE2 configuration, on the other hand, should be used for time series exposures, for which some steps are set to be skipped by default. Both configurations call the Image2Pipeline; the only difference is which steps are applied.\n",
```

Comment:
You might want to define "configuration" here, or perhaps re-word to indicate that there are different sets of pre-defined steps that are mode dependent.

Reply:
Fixed the statement